build.sbt

```diff
@@ -49,7 +49,8 @@ libraryDependencies ++= Seq(
   "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % jacksonVersion,
   "com.groupon.dse" % "spark-metrics" % "2.0.0",
   "org.apache.commons" % "commons-text" % "1.6",
-  "org.influxdb" % "influxdb-java" % "2.14"
+  "org.influxdb" % "influxdb-java" % "2.14",
+  "org.elasticsearch" % "elasticsearch-hadoop" % "6.2.1"
```

Comment:
We can't have this in the sbt file, as it will pin us to a specific version. The correct package needs to be supplied at runtime with ```--packages``` or by adding it to the classpath.